src/database/all.sql

```diff
@@ -192,19 +200,77 @@ CREATE TABLE request (
      FOREIGN KEY (client_id) REFERENCES client(client_id),
      FOREIGN KEY (professional_id) REFERENCES client(professional_id),
      FOREIGN KEY (service_id) REFERENCES service(service_id)
-);CREATE TABLE request_subscription (
-	 request_id            MEDIUMINT NOT NULL,
+);  
+  
+------------------------------ 
+-- From: 160_request_subscription.sql 
+CREATE TABLE request_subscription (
+     request_id            MEDIUMINT NOT NULL,
      subscription_id       MEDIUMINT NOT NULL,
      PRIMARY KEY (request_id, subscription_id),
      FOREIGN KEY (request_id) REFERENCES request(request_id),
      FOREIGN KEY (subscription_id) REFERENCES subscription(subscription_id)
-);
-
+);  
+  
+------------------------------ 
+-- From: 170_request_transcation.sql 
 CREATE TABLE request_transaction (
-	 request_id           MEDIUMINT NOT NULL,
+     request_id           MEDIUMINT NOT NULL,
      transaction_id       MEDIUMINT NOT NULL,
      PRIMARY KEY (request_id, transaction_id),
      FOREIGN KEY (request_id) REFERENCES request(request_id),
      FOREIGN KEY (transaction_id) REFERENCES transaction(transaction_id)
-);
+);  
+  
+------------------------------ 
+-- From: 300_insert_billing_type.sql 
+INSERT INTO billing_type (billing_type_name,retired) VALUES ('Out', null);
+
+INSERT INTO billing_type (billing_type_name,retired) VALUES ('In', null);
+
+COMMIT;  
+  
+------------------------------ 
+-- From: 310_insert_subscriptions.sql 
+INSERT INTO subscription (subscription_name, fee) VALUES ('Subscription', 100.50);
+
+INSERT INTO subscription (subscription_name, fee) VALUES ('Single', 0);
+
+COMMIT;  
+  
+------------------------------ 
+-- From: 320_insert_service.sql 
```

Comment:
Hey Carl, please make sure that the below fields are in the database, keeping exactly to what is written below. I have no issues if you wanted to included others but please make sure the below are inserted exactly as they are listed in this comment.

Tree Removal
Roof Cleaning
Fence Installation
Plumbing
Oven Repairs